<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-UVVSPQO" failures="1" tests="2" name="Default test(failed)" time="0.095" errors="0" timestamp="2022-05-04T14:47:11 IST">
  <testcase classname="testngparallelexecution.FailedTestCaseExecution" name="login" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testngparallelexecution.FailedTestCaseExecution.login(FailedTestCaseExecution.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testngparallelexecution.FailedTestCaseExecution" name="profile" time="0.0">
    <skipped/>
  </testcase> <!-- profile -->
  <testcase classname="testngparallelexecution.FailedTestCaseExecution" name="dashbord" time="0.0">
    <ignored/>
  </testcase> <!-- dashbord -->
  <testcase classname="testngparallelexecution.FailedTestCaseExecution" name="logout" time="0.0">
    <ignored/>
  </testcase> <!-- logout -->
</testsuite> <!-- Default test(failed) -->
